docs/docker.md

```diff
@@ -72,6 +72,51 @@ E.g., to start a shell in a container started with `--name pl`:
 docker exec -it pl /bin/bash
 ```
 
+## Docker-Compose
+
+### Basics
+
+A docker-compose file describes the services an application needs to run. In our case, we use `docker-compose` to configure and run the PrairieLearn docker container locally.
+
+To run PrairieLearn with `docker-compose`, run `docker-compose up pl`. This will, in order:
```

Comment:
You can use a numbered list to denote the sequential nature of the steps.

Reply:
I don't know if search engines read markdown syntax for semantics yet, but I wouldn't be surprised if they do. They do read semantics in HTML.